python/test_file_generator.py

```diff
@@ -2654,32 +2569,165 @@ def v705_2(self):
+		lar.co_app_credit_score = "650"
+		lar.app_score_name = "1"
+		lar.co_app_score_name = "1"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v710_2(self):
+		"""Set Applicant Credit Score to Exempt, and set
+		Applicant Score Name, Co-Applicant Score Name, Applicant Score Code 8
+		and Co-Applicant Score Code 8 to 8, 8, "New Scoring Model", and "New Scoring Model" 
+		respectively."""
+		name = "v710_2.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.app_credit_score = "1111"
+		lar.app_score_name = "8"
+		lar.co_app_score_name = "8"
+		lar.app_score_code_8 = "New Scoring Model"
+		lar.co_app_score_code_8 = "New Scoring Model"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v711(self):
+		"""Set Denial 1 to Exempt, and set
+		Denial 2, Denial 3, Denial 4
+		and Denial Code 9 to 2, 3, 4, 
+		respectively."""
+		name = "v711.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.denial_1 = "1111"
+		lar.denial_2 = "2"
+		lar.denial_3 = "3"
+		lar.denial_4 = "4"
+		lar.denial_code_9 = ""
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v712(self):
+		"""Set Loan Costs to Exempt, and set
+		Points and Fees to 1000."""
+		name = "v712.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.loan_costs = "Exempt"
+		lar.points_fees = "1000"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+
+	def v713_1(self):
+		"""Set AUS 1 to Exempt, and AUS Result 1 to 1."""
+		name = "v713_1.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.aus_1 = "1111"
+		lar.aus_result_1 = "1"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+	
+	def v713_2(self):
+		"""Set AUS 1 to Exempt, AUS 2-5, AUS Result 2-5, 
+		AUS Code 5, and AUS Code 16 to 1."""
+		name = "v713_2.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.aus_1 = "1111"
+		lar.aus_2 = "1"
+		lar.aus_3 = "1"
+		lar.aus_4 = "1"
+		lar.aus_5 = "1"
+		lar.aus_result_1 = "1"
+		lar.aus_result_2 = "1"
+		lar.aus_result_3 = "1"
+		lar.aus_result_4 = "1"
+		lar.aus_result_5 = "1"
+		lar.aus_code_5 = "1"
+		lar.aus_code_16 = "1"
+		print("writing {name}".format(name=name))
+		utils.write_file(name=name, path=path, ts_input=ts, lar_input=lar)
+		
+	def v714_const(self):
+		"""Set Submission of Application to Exempt and Initially Payable
+			to Your Institution to 1."""
+		name = "v714.txt"
+		path = self.validity_path
+		ts = self.ts_df.copy()
+		lar = self.lar_df.copy()
+		lar.app_submission = "1111"
+		lar.initially_payable= "1"
```

Comment:
In order to test both cases there should either be two files or the file should check conditions where initially_payable = '1111' and app_submission != '1111'